internal/mode/static/nginx/runtime/manager.go

```diff
@@ -22,7 +22,7 @@ const (
 	// PidFile specifies the location of the PID file for the Nginx process
 	PidFile = "/var/run/nginx/nginx.pid"
 	// pidFileTimeout defines the timeout duration for accessing the PID file
-	pidFileTimeout = 10000 * time.Millisecond
+	PidFileTimeout = 10000 * time.Millisecond
```

Comment:
@kate-osborn exported as I needed this in another file here, https://github.com/nginxinc/nginx-gateway-fabric/pull/2175/commits/d01c93ef915e15fb6bba68056aa54a543348cb98#diff-555c45daf604f58134a5937da621ca2d0c8900c924dfd21fabd417957ace5c82R127